ui-v2/tests/steps.js

```diff
@@ -5,6 +5,8 @@ import getDictionary from '@hashicorp/ember-cli-api-double/dictionary';
 import pages from 'consul-ui/tests/pages';
 import api from 'consul-ui/tests/helpers/api';
 
+// const dont = `( don't| shouldn't| can't)?`;
```

Comment:
Is this comment still useful?

Reply:
Yup, that's coming in soon, in that 'on the horizon steps PR'. Just commented it out here as I got it all working in whilst playing in this branch just didn't want it to go in on this branch.